delfin/api/schemas/storage_capabilities_schema.py

```diff
@@ -76,7 +76,7 @@
                                                 'maxLength': 255}
                             },
                         },
-                        'writeRequests': {
+                        'writeIops': {
```

Comment:
Is these metric names are common metric names? I see it is changed now. What is the possibility of further change? Can we use any key mapping to keep the code more maintainable?

Reply:
applicable for all

Reply:
Yes These are common metric names, based on our analysis on different platforms we derived common names now. hence to align we have changed this metric names. This is schema would be validated against driver capabilities.. drivers would use their mapping to map storage metric names  with delfin  metric names